101-traffic-manager-external-endpoint/azuredeploy.json

```diff
@@ -0,0 +1,58 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "dnsname": {
```

Comment:
That's a fair suggestion, I'm trying to keep the names as similar to the portal as possible, where we call the field DNS Name.